mixin/main.go

```diff
@@ -228,12 +228,46 @@ func AddPackage(pkg, bundle string, build bool) error {
 		return err
 	}
 
-	if !build {
-		return nil
+	if build {
+		return buildMix(false)
 	}
 
+	return nil
+}
+
+func buildMix(prepNeeded bool) error {
+	var err error
 	lastVer := getLastVersion()
+	ver, err := getCurrentVersion()
+	if err != nil {
+		return err
+	}
+	mixVer := ver * 1000
 	oldMix := filepath.Join(mixWS, fmt.Sprintf("update/www/%d", mixVer-10))
+	b, err := builder.NewFromConfig(filepath.Join(mixWS, "builder.conf"))
```

Comment:
I would personally change this to mixWorkspace for clarity, ymmv :nerd_face:

Reply:
I prefer `mixWS` for brevity's sake and because in the context `mixWS` is pretty clear.